File: 프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.cs
using System;

public class Solution 
{
    public int solution(string my_string) 
    {
        int hideNumSum = 0;
        for(int i=0; i<my_string.Length; i++)
        {
            if(Char.IsDigit(my_string[i]))
            {
                hideNumSum += Int32.Parse(my_string[i].ToString());
            }
        }
        return hideNumSum;
    }
}

File: 프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string my_string)
{
    vector<char> vec;
    for(char &c : my_string)
    {
        if(isdigit(c))
        {
            vec.push_back(c);
        }
    }
    
    int sum = 0;
    for(char &c : vec)
    {
        sum += c - '0';
    }
    return sum;
}